src/Model/Model.ts

```diff
@@ -550,6 +550,7 @@ export class BaseModel {
       $isDeleted: this.$isDeleted,
       $dirty: this.$dirty,
       $isDirty: this.$isDirty,
+      $isTransaction: Boolean(this.$trx),
```

Comment:
I'd only add this if `$isTransaction` exists on the class. Otherwise it's a bit confusing because all other props starting with `$` exist

Reply:
Ok, I'll add this getter in model